src/types/catalog.d.ts

```diff
@@ -16,13 +14,6 @@ export interface ShopperCatalogResource<T> extends Resource<T> {
   }
 }
 
-export interface ShopperCatalogResourceList<T> extends ResourceList<T> {
```

Comment:
Maybe we should flag this as deprecated and remove it in the next major version @samblacklock @butikden 

It's been in there long enough that someone could be using the type 

```
/**
* @deprecated use ShopperCatalogResourcePage instead. Will be removed on next major release.
*/
export interface ShopperCatalogResourceList<T> extends ResourceList<T> {
```

Reply:
Makes sense - let's make a mental note to follow up with the removal in the next `MAJOR`